ml3d/torch/dataloaders/default_batcher.py

```diff
@@ -1,7 +1,11 @@
-import torch
 import re
-from torch._six import container_abcs, string_classes, int_classes
+import collections
```

Comment:
This change is just to say more Pythonic. Low level imports on the top, and then external libraries like `torch`.